xds/internal/httpfilter/rbac/rbac.go

```diff
@@ -126,7 +126,10 @@ func parseConfig(rbacCfg *rpb.RBAC) (httpfilter.FilterConfig, error) {
 		return config{}, nil
 	}
 
-	ce, err := rbac.NewChainEngine([]*v3rbacpb.RBAC{rbacCfg.GetRules()})
+	// TODO - change the call chain to here so we have the filter name to input
```

Comment:
Should this be tracked in a GH issue?

Reply:
I can add that if it's good for your workflow - I'm tracking it personally because it's one of the next things I'm going to do.

Reply:
Works! Optional: prolly just own the TODO like
```suggestion
	// TODO(gregorycooke) - change the call chain to here so we have the filter name to input
```